aapp_runner/tle_satpos_prepare.py

```diff
@@ -200,24 +231,31 @@ def do_tleing(config, timestamp, satellite):
 
     _ensure_tledir(DIR_DATA_TLE)
 
+    # Fetch TLE files from central real-time repo and place them under the AAPP orbelems structure:
+    if 'recent_tlefiles_ext_dir' in config['aapp_processes'][config.process_name]:
```

Comment:
This new config isn't documented anywhere. Please add it to the documentation and config examples.

Reply:
I have added an explanation in the template config file.